bin/jeprof.in

```diff
@@ -3083,6 +3101,8 @@ sub RemoveUninterestingFrames {
     foreach my $a (@addrs) {
       if (exists($symbols->{$a})) {
         my $func = $symbols->{$a}->[0];
+        # Remove suffix in the symbols following space when filtering.
```

Comment:
@guangli-dai can you share an example of what this filters out?

Reply:
Here it is:

```
   ...
   165.4   2.0%  97.9%    165.4   2.0% prof_alloc_prep (inline)
   ...
```

This is the heap profile dumped from a python program.

Reply:
Is this to filter out the `(inline)` suffix? Does it affect any case that doesn't have the inline part?

Reply:
Yes, it is. Since this simply remove the suffix after a whitespace (" "), cases without such suffix will stay exactly the same.